HTTPlayground/main.py

```diff
@@ -0,0 +1,39 @@
+from http.server import HTTPServer, BaseHTTPRequestHandler
+from HTTPlayground.views import not_found
+from HTTPlayground.urls import urls
+from HTTPlayground.settings import URL, PORT
+from HTTPlayground.base import FileReader, send_headers
+
+
+class SimpleRequestHandler(BaseHTTPRequestHandler):
+
+    def do_GET(self):
```

Comment:
try `curl -X POST http://localhost:8000/form` - it draws your a form, even if it's a POST request.
This means you hardly want to use the same `urls` for all HTTP methods.

Reply:
You offer separate urls on GET_urls and POST_urls?

> try `curl -X POST http://localhost:8000/form` - it draws your a form, even if it's a POST request.
> This means you hardly want to use the same `urls` for all HTTP methods.

Reply:
maybe. The main point is that there are not only urls. There are also methods that are available for certain urls.